lib/tests/test.js

```diff
@@ -120,6 +120,7 @@ class Test {
       ipcRole: 'client'
     });
     this.events.request('deploy:setGasLimit', 6000000);
+    this.engine.startService("codeCoverage");
```

Comment:
Picky comment. Can you move this one line up. That way all service starts are in one block.